lib/active_admin/application.rb

```diff
@@ -250,6 +256,8 @@ def controllers_for_filters
       controllers
     end
 
+    attr_writer :routes_reloader # :nodoc:
```

Comment:
Alternatively we could return the capture the reloader with `@routes_reloader = attach_reloader` in `prepare!`. That will just force a local variable capture into `attach_reloader`. Let me know which you prefer 😁